src/functionalTest/java/uk/gov/hmcts/reform/bulkscanprocessor/controllers/TestHelper.java

```diff
@@ -66,7 +67,10 @@ public void uploadZipFile(
     }
 
     public void uploadZipFile(
-        CloudBlobContainer container, List<String> files, String metadataFile, final String destZipFilename
+        CloudBlobContainer container,
+        List<String> files,
+        String metadataFile,
+        final String destZipFilename
```

Comment:
is this final really required here ?

Reply:
Probably everything is final there. Just made a change to retrigger build with `[PREVIEW]`